### DISCUSSION TITLE
https://eips.ethlibrary.io/zh/eip-1559.html

### DISCUSSION THREAD

**Discussion by giscus** [ethlibrary/EIPs#20] [Announcements]
# https://eips.ethlibrary.io/zh/eip-1559.html

以太坊改进提案（EIPs）描述了以太坊平台的标准，包括核心协议规范，客户端API和合约标准。

https://eips.ethlibrary.io/zh/eip-1559.html

**Comment by izayl** [comment]
![](https://camo.githubusercontent.com/e35921e2d0900268f4294cca4037e786b061a111f3fa4ffc9ac2d3fa754cf4aa/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4d54636c6643723474566769732f67697068792e676966)
